frontend/src/widgets/OverviewWidgetField.scss

```diff
@@ -0,0 +1,51 @@
+@media( min-width: 1024px ){
+    .smart-hub--dashboard-overview-widget-field {
+        margin: 0;
+    }
+}
+
+.smart-hub--dashboard-overview-widget-field-icon-background-lg {
+    border-radius: 50%;
+    height: 5em;
+    width: 5em;
+}
+
+.smart-hub--dashboard-overview-widget-field-icon-background-sm {
+    border-radius: 50%;
+    height: 3em;
+    width: 3em;
+}
+
+.smart-hub--overview-widget-field-font-size {
+    font-size: 15px;
+}
+
+.smart-hub--overview-widget-field-data-font-size {
+    font-size: 19px;
+}
```

Comment:
Can we use the uswds font size classes for these, please?

Reply:
any luck on this change ^ @AdamAdHocTeam

Reply:
@thewatermethod oops missed this looking now

Reply:
should be updated now.